src/pages/Books/Books.tsx

```diff
@@ -61,9 +69,14 @@ export const Books: React.FC = () => {
       searchQuery={searchQuery}
       setSearchQuery={setSearchQuery}
     >
-      {isLoading ? (
-        <Loading message="loading books" />
-      ) : (
+      {searchQuery == null && (
+        <div className={style.welcome}>
+          <img src={search} className={style.welcomeImage} />
+        </div>
+      )}
```

Comment:
Is it intentional, that the initial state is different?

![CleanShot 2024-12-23 at 00 05 09@2x](https://github.com/user-attachments/assets/43ddccf6-a3b0-4626-93fe-4b8c77de7ca8)

vs typing and deleting all?

![CleanShot 2024-12-23 at 00 07 11](https://github.com/user-attachments/assets/4a476b00-0077-4ea5-acb2-8c021eeb3d34)